galpy/potential/potential_c_ext/DehnenPotential.c

```diff
@@ -0,0 +1,113 @@
+#include <math.h>
+#include <gsl/gsl_sf_gamma.h>
+#include "sf_math.h"
+#include "galpy_potentials.h"
+//DehnenPotential
+//3 arguments: amp, a, alpha
+double DehnenSphericalPotentialEval(double R,double Z, double phi,
+			      double t,
+			      struct potentialArg * potentialArgs){
+  double * args= potentialArgs->args;
+  double amp= *args++;
+  double a= *args++;
+  double alpha= *args;
+  // See if Hernquist or Jaffe
+  if (alpha == 1.) {
+    return HernquistPotentialEval(R, Z, phi, t, potentialArgs);
+  }
+  else if (alpha == 2.) {
+    return JaffePotentialEval(R, Z, phi, t, potentialArgs);
+  }
+  // else, calculate Rforce
```

Comment:
As long as this code is run through galpy's Python interface (which I think we should assume), I don't think the user could ever end up here, because Hernquist and Jaffe should just be caught at the potential's initialization. So I think we should remove these special cases in this file.

Reply:
first just commenting them out to test.